docs/use-cases/attachments.md

```diff
@@ -16,7 +16,7 @@ const msg = {
       filename: 'some-attachment.txt',
       type: 'plain/text',
       disposition: 'attachment',
-      content_id: 'mytext'
+      contentId: 'mytext'
```

Comment:
As per open api specification, it is [content_id](https://github.com/twilio/sendgrid-oai/blob/main/spec/yaml/tsg_mail_v3.yaml#L601)